lua/lazyvim/config/keymaps.lua

```diff
@@ -129,6 +129,20 @@ map("n", "<leader>ub", function() Util.toggle("background", false, {"light", "da
 map("n", "<leader>gg", function() Util.terminal({ "lazygit" }, { cwd = Util.root(), esc_esc = false, ctrl_hjkl = false }) end, { desc = "Lazygit (root dir)" })
 map("n", "<leader>gG", function() Util.terminal({ "lazygit" }, {esc_esc = false, ctrl_hjkl = false}) end, { desc = "Lazygit (cwd)" })
 
+-- Function to get the current file path from the root directory
+local current_file_path_from_root_dir = function()
+  local path = vim.fn.expand("%:p")
+  local root = Util.root()
+  if not path:find(root, 1, true) then
+    return path
+  end
+  return path:sub(#root + 2)
+end
```

Comment:
This may not get the proper path if the lsp gives a different root than where the git repo is.

Is there a better way to do this (i.e. get the path to the file from the git root)?